madprops/serializers.py

```diff
@@ -0,0 +1,138 @@
+from collections import Iterable
+
+from django.utils.functional import cached_property
+from rest_framework.serializers import (
+    ModelSerializer, ModelSerializerOptions, RelationsList)
+
+
+class PropertiesSerializerOptions(ModelSerializerOptions):
+    """Meta class options for PropertiesSerializer"""
+    def __init__(self, meta):
+        super(PropertiesSerializerOptions, self).__init__(meta)
+        self.parent_obj_field = getattr(meta, 'parent_obj_field', None)
```

Comment:
I think we can make serializer smart and guess this field automatically (from StackOverflow):
```
for field in ModelC._meta.fields:
    if field.get_internal_type() == "ForeignKey":
        print field.rel.to
```